connector/src/main/java/com/google/cloud/bigquery/connector/common/BigQueryReadClientFactory.java

```diff
@@ -15,22 +15,29 @@
  */
 package com.google.cloud.bigquery.connector.common;
 
+import com.esotericsoftware.minlog.Log;
```

Comment:
I don't see it used in the code. Why do we need an additional logger?

Reply:
removed.  intellij be only open.